input/pagecontent/use-cases.md

```diff
@@ -1,7 +1,10 @@
 ## AU eRequesting R1 use cases
```

Comment:
1. This PR is final PR for use case material for this IG. That means any and all changes to improve / define what is needed will be part of the scope of this PR.